.gitlab-ci.yml

```diff
@@ -6,12 +6,12 @@ stages:
 variables:
   DOCKER_DRIVER: overlay2
   DOCKER_REGISTRY: ${CI_REGISTRY}
-  # Docker image name to use when building base image
-  CONTAINER_IMAGE_BASE: ${CI_REGISTRY_IMAGE}:${CI_COMMIT_REF_SLUG}-${CI_BUILD_REF}
-  # Docker image name to use when building apache image
-  CONTAINER_IMAGE_RELEASE: ${CI_REGISTRY_IMAGE}:${CI_COMMIT_REF_SLUG}-${CI_BUILD_REF}-apache
+  # Docker image name to use when building
+  CONTAINER_IMAGE: ${CI_REGISTRY_IMAGE}:${CI_COMMIT_REF_SLUG}-${CI_BUILD_REF}
   # Git Repo of ansible playbooks
   PLAYBOOK_REPO_URL: https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.dhe.duke.edu/gcb-informatics/gcb-ansible-cd.git
+  # Playbook branch or ref to checkout
+  PLAYBOOK_REF: reorg-d4s2-docker
```

Comment:
Does `PLAYBOOK_REF: reorg-d4s2-docker` need to be set to something else just before commit?